content/packages.md

```diff
@@ -31,11 +32,124 @@ my-package
 
 The `package.js` file is the main file in every Meteor package. This is a JavaScript file that defines the metadata, files loaded, architectures, NPM packages, and Cordova packages for your Meteor package.
 
-[Read about the `package.js` API in the Meteor docs.](http://docs.meteor.com/#/full/packagejs)
+In this guide article, we will go over some important points for building packages, but we won't explain every part of the `package.js` API. To learn about all of the options, [read about the `package.js` API in the Meteor docs.](http://docs.meteor.com/#/full/packagejs)
+
+## onUse vs. onTest
+
+XXX
+
+## Adding files and assets
+
+The main function of a Meteor package is to contain source code (JS, CSS, and any transpiled languages) and assets (images, fonts, and more). They act as a unit of code organization, give the developer complete control over which files are loaded where, and can be shared between different applications. Here's how you add files to a package:
+
+```js
+// Example for adding files to a package
+// Pulled from the lists-show package in the Todos example app
+Package.onUse(function(api) {
+  // ...
+
+  api.addFiles([
+    'todos-item.html',
+    'todos-item.js',
+    'lists-show.html',
+    'lists-show.js',
+    'lists-show-page.html',
+    'lists-show-page.js',
+    'lists-show.less'
+  ], 'client');
+
+  // ...
+});
+```
+
+And the same for assets:
+
+```js
+// Example for adding assets to a package
+// Pulled from the less-imports package in the Todos example app
+Package.onUse(function(api) {
+  // ...
+
+  api.addAssets([
+    'font/OpenSans-Light-webfont.eot',
+    'font/OpenSans-Light-webfont.svg',
+    'font/OpenSans-Light-webfont.ttf',
+    'font/OpenSans-Light-webfont.woff',
+    'font/OpenSans-Regular-webfont.eot',
+    'font/OpenSans-Regular-webfont.svg',
+    'font/OpenSans-Regular-webfont.ttf',
+    'font/OpenSans-Regular-webfont.woff',
+  ], 'client');
+
+  // ...
+});
+```
+
+As you can see, the `addFiles` and `addAssets` functions allow you to pass a list of files as the first argument, and an architecture (or array of architectures) as the second argument. For more information about this, see the section about architectures below.
+
+## Package dependencies
+
+Another very important feature of Meteor packages is the ability to register dependencies on other packages. This is done via `api.use` and `api.imply`.
```

Comment:
I would add something about `api.exports()` first so it's clear what the end effect of `use()` and `imply()` is